src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandler.java

```diff
@@ -109,9 +109,8 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                 sscsCaseData.setProcessingVenue(processingVenue);
             }
         }
-        if (adjournment.getGeneratedDate() == null) {
-            adjournment.setGeneratedDate(LocalDate.now());
-        }
+
+        adjournment.setGeneratedDate(LocalDate.now());
```

Comment:
Is it possible to add a test, where you set the adjournment.setGeneratedDate(Yesterday);

then you trigger the write adjournment notice event again and check if the GeneratedDate is updated to todays date.

Reply:
added the new unit test and pipline is green